oidc-controller/api/main.py

```diff
@@ -132,13 +134,22 @@ async def on_tenant_startup():
     """Register any events we need to respond to."""
     await init_db()
     await init_provider(await get_db())
+
+    # Start the APScheduler-based cleanup service
+    await cleanup_service.start_scheduler()
+    logger.info("Started APScheduler-based presentation record cleanup service")
```

Comment:
This depends on Redis, we should NOT attempt to start this up if Redis is not being used.
Probably best to handle this inside the `start_scheduler` function so we can log success/failure depending on a self-check.